src/platforms/javascript/guides/capacitor/ionic.mdx

```diff
@@ -4,4 +4,8 @@ sidebar_order: 201
 description: "Learn how to use Sentry with Ionic."
 ---
 
-Sentry Capacitor supports [Ionic](https://ionicframework.com/), check the [Getting Started documentation](/platforms/javascript/guides/capacitor/) to setup your Ionic project with Sentry Capacitor SDK.
\ No newline at end of file
+Sentry Capacitor supports [Ionic](https://ionicframework.com/) out of the box, the documentation below is similar to the one at [Getting Started documentation](/platforms/javascript/guides/capacitor/) since the setup of both Capacitor and Ionic are the same without the need of any additional change.
```

Comment:
Fixed some punctuation and wording. 

```suggestion
Sentry Capacitor supports [Ionic](https://ionicframework.com/) out of the box. The documentation below is similar to our [Capacitor Getting Started documentation](/platforms/javascript/guides/capacitor/) since the setup of Capacitor and Ionic are the same.
```